src/main/java/mate/academy/hw01/Matrix.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.hw01;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Matrix {
+    static Integer[][] arr;
+
+    public static void main(String[] args) {
+        arr = createArray(3, 3);
+        System.out.println(Arrays.deepToString(arr));
+        foundMinMax(arr);
+    }
+
+    private static Integer[][] createArray(Integer m, Integer n) {
+        Integer[][] array = new Integer[m][n];
+        Random random = new Random();
+        for (int i = 0; i < m; i++) {
+            for (int j = 0; j < n; j++) {
+                array[i][j] = random.nextInt(100);
+            }
+        }
+        return array;
+    }
+
+    private static void foundMinMax(Integer[][] array) {
```

Comment:
find not found

Reply:
This method finds the minimum and maximum elements of the matrix, I checked

Reply:
I changed